security/pkg/nodeagent/test/cert_rotation/doc.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rotatesds
```

Comment:
why do we need these empty files?

Reply:
Jimmy told me golang doesn't allow all files as "_test" files in a directory, and telemetry is doing this.

Reply:
I follow the tests in mixer/test/client/.... If I recall correctly, the doc.go was added there because golang build complains folder with only test file.